data/hai/hai reveal 1 intro.txt

```diff
@@ -11,47 +11,835 @@
+
+
+
+mission "Hai Reveal Concern Period: Reactivating Sandsnap"
+	priority
+	name "Transport the Stationkeeper to <planet>"
+	description "Safely bring the Stationkeeper to <destination> to reactivate the station."
+	source "Makerplace"
+	destination "Sandsnap"
+	passengers 1
+	to offer
+		has "HR: Call for Aid with Sandsnap: active"
+	blocked `You are advised that participation in the Sandsnap Project will require at least 1 bunk free for passengers.`
+	on offer
+		conversation
+			`	Entering the spaceport requires one to walk around a very large holosign giving directions to the Sandsnap Project's office.`
+			choice
+				`	(Check out the project.)`
+				`	(Come back later.)`
+					defer
+				`	(Ignore it, you've changed your mind.)`
+					decline
+			`	There is a greeter, or perhaps a security guard, at the door who appraises you as you approach. "You must be here for the project," he says. "Identification please."`
+			`	After a quick check of your ID he gives you a very professional Hai smile. "Welcome to the Sandsnap Project. Your credentials are sufficient to apply so please, do come in. We hope you'll find the project interesting and the effort rewarding."`
+			`	As you enter a harried Hai gestures you over with enthusiasm. "Greetings," he checks his tablet. "Captain <last>! Your presence in these dire times is greatly appreciated - it has been at least a hundred years since we have had such a threat to our way of life! You must have noticed the spike in pirate activity. Perhaps our northern brethren have had a point all these years."`
+			`	He stops himself. "Not that our politics probably concern you. Let us cut to the matter at hand, shall we?"`
+			choice
+				`	(Nod.)`
+					goto border
+				`	"What are you talking about?"`
+				`	"Not at all, I love politics."`
+			`	He seems surprised. "Oh. Well in that case, the Unfettered have been militarizing a lot in the last 20 years, forcing us to do the same in order to protect merchants from their raids. They may be right that we have grown weak, if the success of those pirate raids is any indication; security forces were simply not ready for them. We here are in the business of ensuring this will not happen again. Mark my words, nothing like an arms manufacturer setting up a support base to really wake a few people up.`
+			label border
+			`	"The raids have proven that we are not prepared to ensure true security on our borders. In order to address this we want to reactivate the mothballed station of <destination>, for which we have acquired the relevant paperwork and access codes. We have already hired a few mercenaries to help with the defense of our station, mostly local humans with experienced pasts. Once operational, the station should be a major strategic advantage for us, allowing resupply and repairs of ships at a location suitable for facilitating rapid response to detected threats."`
```

Comment:
```suggestion
			`	"The raids have proven that we are not prepared to ensure true security on our borders. In order to address this we want to reactivate the mothballed station of <planet>, for which we have acquired the relevant paperwork and access codes. We have already hired a few mercenaries to help with the defense of our station, mostly local humans with experienced pasts. Once operational, the station should be a major strategic advantage for us, allowing resupply and repairs of ships at a location suitable for facilitating a rapid response to detected threats."`
```

Reply:
Because it's a mothballed station that you're not expected to have visited, the full destination is used intentionally.

Reply:
That makes sense. Feel free to mark this as resolved.